src/scalarstats.jl

```diff
@@ -153,9 +182,21 @@ end
 #
 #############################
 
+"""
+    percentile(v, p)
+
+Return the `p`th percentile of a real-valued array `v`.
+"""
 percentile{T<:Real}(v::AbstractArray{T}, p) = quantile(v, p * 0.01)
 
 quantile{T<:Real}(v::AbstractArray{T}) = quantile(v, [.0, .25, .5, .75, 1.0])
+
+"""
+    nquantile(v, n)
+
+Return the n-quantiles of a real-valued array, i.e. the values which
+partition `v` into `n` subsets of nearly equal size.
```

Comment:
As I said above, I don't think `nquantile` does that. If you just call it as `nquantile(x)`, it's a `MethodError`.